### ISSUE TITLE
how to use sample weight with CRF as sample weight is a 2D array?

### ISSUE THREAD

**Issue by ShamilaNasreen** [luozhouyang/keras-crf#9]
I am building a model for sequence tagging using BiLSTM and CRF. I am using the Keras-CRF provided in this project. I successfully run the code and below is the summary of my model. 
While the Keras_model support sample weight as 1D but we have to assign different weights at each time step as I want to give higher weights to less frequent tag class. so my weight_sample shape is (1114,100).  It gives error as to compute the loss the function is:
if sample_weight is not None:
                crf_loss = crf_loss * sample_weight
and an error occued of 'required broadcast shape'.
![image](https://user-images.githubusercontent.com/45317868/149960079-44e6342d-50b4-43ee-a6a6-741a5f201c4d.png)

Can you please suggest how to use 2D sample weight within this code.
